### PR TITLE
mcs: collect scheduling statistics

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Issue Number: Ref #5839.

### What is changed and how does it work?

This PR collects the statistics on the scheduling service and modifies Grafana template to make the monitor work as before.

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by JmPotato — APPROVED** [review]

File: metrics/grafana/pd.json [hunk 1]
```diff
@@ -1663,7 +1663,7 @@
           "tableColumn": "idalloc",
           "targets": [
             {
-              "expr": "pd_cluster_id{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=\"$instance\", type=\"idalloc\"}",
+              "expr": "max(pd_cluster_id{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", type=\"idalloc\"})",
```
Comment (rleungx) [inline metrics/grafana/pd.json:1666]: change to max

File: metrics/grafana/pd.json [hunk 2]
```diff
@@ -2202,7 +2202,7 @@
           "tableColumn": "tso",
           "targets": [
             {
-              "expr": "pd_cluster_tso{instance=\"$instance\", type=\"tso\", dc=\"global\"}",
+              "expr": "max(pd_cluster_tso{type=\"tso\", dc=\"global\"})",
```
Comment (rleungx) [inline metrics/grafana/pd.json:2205]: change to max

File: metrics/grafana/pd.json [hunk 3]
```diff
@@ -2508,7 +2508,7 @@
...
```
Comment (rleungx) [inline metrics/grafana/pd.json:2511]: change to max

**Review by lhy1024 — APPROVED** [review]

**Review by JmPotato — APPROVED** [review]

**Pull request merged**